src/Microsoft.ML.TimeSeries/IidChangePointDetector.cs

```diff
@@ -33,12 +32,12 @@ namespace Microsoft.ML.TimeSeriesProcessing
     /// <summary>
     /// This class implements the change point detector transform for an i.i.d. sequence based on adaptive kernel density estimation and martingales.
     /// </summary>
-    public sealed class IidChangePointDetector : IidAnomalyDetectionBase
+    public sealed class IidChangePointDetector : IidAnomalyDetectionBaseWrapper, IStatefulTransformer, ICanSaveModel
     {
         internal const string Summary = "This transform detects the change-points in an i.i.d. sequence using adaptive kernel density estimation and martingales.";
-        public const string LoaderSignature = "IidChangePointDetector";
-        public const string UserName = "IID Change Point Detection";
-        public const string ShortName = "ichgpnt";
+        internal const string LoaderSignature = "IidChangePointDetector";
+        internal const string UserName = "IID Change Point Detection";
+        internal const string ShortName = "ichgpnt";
 
         public sealed class Arguments : TransformInputBase
```

Comment:
>Arguments [](start = 28, length = 9)

Can you rename to `Options` and make it internal, once the constructor of the estimator is made internal this should be possible.